docs/web/diff.md

```diff
@@ -0,0 +1,903 @@
+
+* Show outstanding reports that appeared from first analysis to the next (via `--new`):
+  ```sh
+  CodeChecker cmd diff --basename ./test_report_dir --newname ./test_report_dir_src_suppressed/ --new
+  ```
+  Expected output:
+  ```
+  ----==== Checker Statistics ====----
+  ---------------------------------------------------
+  Checker name         | Severity | Number of reports
+  ---------------------------------------------------
+  core.NullDereference | HIGH     |                 1
+  ---------------------------------------------------
+  ----=================----
+  ```
+* Show outstanding reports that disappeared from first analysis to the next (via `--resolved`): 
+  ```sh
+  CodeChecker cmd diff --basename ./test_report_dir --newname ./test_report_dir_src_suppressed/ --resolved
+  ```
+  Expected output:
+  ```
+  ----==== Checker Statistics ====----
+  --------------------------------------------------
+  Checker name        | Severity | Number of reports
+  --------------------------------------------------
+  core.DivideZero     | HIGH     |                 1
+  deadcode.DeadStores | LOW      |                 1
+  --------------------------------------------------
+  ----=================----
+  ```
+* Show outstanding reports that remained from first analysis to the next (via `--unresolved`): 
+  ```sh
+  CodeChecker cmd diff --basename ./test_report_dir --newname ./test_report_dir_src_suppressed/ --unresolved
+  ```
+  Expected output (no checkers are seen):
+  ```
+  ----======== Summary ========----
+  ---------------------------------------------
+  Number of processed analyzer result files | 0
+  Number of analyzer reports                | 0
+  ---------------------------------------------
+  ----=================----
+  ```
+
+## Review status rules and diffs
+
+Similarly to source code suppressions, you can set the review status of reports [on the GUI](../..//web/server/vue-cli/src/assets/userguide/userguide.md#review-status) as well, which will create a review status rule. Unlike source code suppressions, review status rules set the review status for _all reports_ matching that rule, _regardless_ whether the rule was created before or after the report was stored. Because these rules are stored on the server, **they play no part in local-local diffs as of yet**.
+
+:warning: Note: Review status rules act differently when runs are compared as opposed to when tags or dates are compared (see [below](#diff-on-tags-or-timestamps)).
+
+:warning: Note: If you are trying out with these examples, keep in mind that review status rules are _not_ deleted even if _all reports that it matches_ (by removing some or all runs in a given product) are. You will need to click on "Configuration" in the upper right corner, select "Review Status Rules", where these rules can be removed.
```

Comment:
Adding an image of this config might make it more understandable for the users.

Reply:
There are pictures in the already linked dics in the same section. You can argue whether the pictures are good enough there, but at least its something :^)